src/styles/colours.scss

```diff
@@ -168,3 +167,19 @@
     box-shadow: 0 1px 0.35rem rgba($dark-grey, 0.5);
   }
 }
+
+.white{
```

Comment:
Maybe instead of having both of these classes, we can just rely on one of them being the default.